action/steps/run_tests.sh

```diff
@@ -82,6 +82,10 @@ if [ $EDIT_MODE = true ]; then
       -runTests \
       -testPlatform editmode \
       -testResults "$FULL_ARTIFACTS_PATH/editmode-results.xml" \
+      -enableCodeCoverage \
+      -coverageResultsPath "$FULL_ARTIFACTS_PATH/CodeCoverage/" \ 
+      -coverageOptions "generateHtmlReport;assemblyFilters:-*unity*" \
+      -burst-disable-compilation
```

Comment:
You're missing a backslash at the end of this line

Reply:
Amended.